abs.yaml

```diff
@@ -0,0 +1,6 @@
+upload_without_merge: true
+aggregate_check: false
+upload_channels:
+  - boldorider4-test-channel
```

Comment:
you probably don't want to use a "reusable" channel like this without a label. this would probably cause conflicts when building different packages, because dependencies are pulled first from the staging channel. without a label any package that fulfils a dependency may be used

Reply:
so you're suggesting to use a fresh channel every time we build concurrent packages?

Reply:
no, just add a label to your channel, like:
```
- boldorider4-test-channel: pytensor
```